tests/rules/import-alias.test.ts

```diff
@@ -355,6 +402,22 @@ function runTests(platform: "win32" | "posix") {
                     },
                 ],
             },
+
+            // relative path overridden for index file
+            {
+                code: `export { Potato } from "./app";`,
+                filename: "src/index.ts",
+                options: [
+                    {
+                        relativeImportOverrides: [
+                            {
+                                pattern: /index.{2,3}$/,
+                                depth: 0,
+                            },
```

Comment:
Might want to include a case where multiple configurations exist as mentioned in one of the comments.

Reply:
i added some and but i think before if there was multiple configs path only the first find were applied